apps/core/scenes/passengerForm/BaggageBundles/BaggageTypeDetail.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Icon } from '@kiwicom/universal-components';
+import type { IconNameType } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { bags, type Bag } from './BagTypes';
+
+type Props = {|
+  +bagType: Bag,
+  +dimensions?: string,
+  +quantity: number,
+  +chosen: boolean,
+  +weight: string,
+|};
+
+function getIcon(bagType, quantity: number): IconNameType {
+  if (quantity === 0 && bagType === bags.PERSONAL_ITEM)
+    return 'baggage-personal-item'; // @TODO add "no personal item icon"
+  if (quantity === 0 && bagType === bags.CHECKED_BAG) return 'close';
+  switch (bagType) {
+    case bags.PERSONAL_ITEM:
+      return 'baggage-personal-item';
+    case bags.CABIN_BAG:
+      return 'baggage-cabin';
+    default:
+      return 'baggage-cabin';
+  }
+}
+
+const BagIcon = ({ bagType, quantity, chosen }) => {
+  let color = defaultTokens.colorIconAttention;
+  if (chosen && quantity === 0 && bagType === bags.PERSONAL_ITEM) {
+    color = defaultTokens.colorIconWarning;
+  } else if (!chosen && quantity === 0) {
+    color = defaultTokens.colorIconSecondary;
+  }
+  return <Icon name={getIcon(bagType, quantity)} color={color} />;
+};
+
+const BagTitle = ({ quantity, bagType, chosen, weight }) => {
```

Comment:
I would suggest to add some tests for components with those conditions.

Reply:
I wanted to add tests later along with missing component's note footer part, cause this PR already getting huge.

Reply:
OK, it makes sense. Could you create issue for it?

Reply:
One for UI and one for the tests?

Reply:
I just added it to TODO inside component issue https://github.com/kiwicom/margarita/issues/122